### PR TITLE
Increase waiting in yast_snapper

### PR DESCRIPTION
**Pull request by asmorodskyi**
Fixing https://openqa.suse.de/tests/846451

### CODE REVIEW

File: tests/x11/yast2_snapper.pm [hunk 1]
```diff
@@ -46,7 +46,7 @@ sub clean_and_quit() {
     # C'l'ose  the snapper module
     wait_screen_change { send_key "alt-l" };
     # Wait until xterm is focussed, delete the directory and close xterm
-    wait_idle;
+    assert_screen 'xterm',        10;
```
Comment (okurz) [inline tests/x11/yast2_snapper.pm]: never use `assert_screen` with less than default timeout unless you are waiting for something which is time-critical itself, e.g. "reboot now" dialog or grub menue with timeout -> just use `assert_screen 'xterm';`

**Comment by okurz** [comment]
how is that supposed to fix? Did you maybe miss https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2672 ? Also in this case when something happens which seems to be very sporadic I guess having some statistics for verification is advised -> clone job 10x for verification? Make use of the weekend on your test server ;-)

**Comment by asmorodskyi** [comment]
Verification run  -  http://kimball.arch.suse.de/tests/322 . also I updated PR with revert of #2672 as it was not the issue

**Comment by asmorodskyi** [comment]
Related PR to sle needles https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/345

File: tests/x11/yast2_snapper.pm [hunk 2]
```diff
@@ -39,15 +39,15 @@ sub y2snapper_create_snapshot() {
...
 sub clean_and_quit() {
     # Ensure yast2-snapper is not busy anymore
     wait_still_screen;
-    # C'l'ose  the snapper module
+    # Close  the snapper module
     wait_screen_change { send_key "alt-l" };
-    # Wait until xterm is focussed, delete the directory and close xterm
-    wait_idle;
-    script_run 'rm -rf testdata', 300;
+    # Wait until root gnome terminal  is focussed, delete the directory and close window
```
Comment (okurz) [inline tests/x11/yast2_snapper.pm]: double w/s

**Pull request merged**